app/src/main/java/grinnell/appdev/edu/lyles/Constants.java

```diff
@@ -8,6 +8,8 @@
 
 public class Constants {
 
+    private Constants() {};
```

Comment:
This constructor is not needed since we only use the `public static final` fields and never instantiate it.